### ISSUE TITLE
confusion about overlay menu options

### ISSUE THREAD

**Issue by arusstaylor** [CARTAvis/carta-frontend#805]
currently we have a menu for "overlay configuration" within the Preference plane. Which allows you to set various options for the AST overlay.  We also have another menu Called "Overlay Settings" which is accessed via "View -> Overlay -> customise which allows access to a much more comprehensive set of options related the AST overlay.  This is confusing.  

I find the overlay configuration menu in the preference to be very limited (e.g. I cannot set the line width of the grid).  We need to rethink this.  I think we should get rid of the view -> overlay -> customise route and just put the full functionality of the overlay settings menu into the overlay configuration in preference.  

More generally the top bar menu needs a rethink. The "view" menu seems to have an unthemed hodge-podge of items, as does the Layout menu.  The layout menu seems to be mostly a list of widgets.  The list of widgets should probably be in the "view" menu, not something called layout. The list of widgets in the layout menu should be in the view menu and layout should probably just have what you currently get with Layout->Layouts so you remove a step to get the Layout management options.

**Comment by veggiesaurus** [comment]
> More generally the top bar menu needs a rethink. The "view" menu seems to have an unthemed hodge-podge of items

I agree. I think we might also want to create a "Frame" or "Image" menu similar to DS9, which will allow users to control spatial and spectral matching.
